### DISCUSSION TITLE
DIPY open lab meetings, Winter 2020

### DISCUSSION THREAD

**Discussion by arokem** [dipy/dipy#2356] [2. Announcements]
Welcome to 2020!

The schedule for upcoming lab meetings is:

January 9th, 0800 PT, 1100 ET, 1600 UTC, Topic: preparations for the March DIPY workshop at IU.
January 23rd, 0800 PT, 1100 ET, 1600 UTC Topic: @arokem on pyAFQ
February 6th, 0800 PT, 1100 ET, 1600 UTC Topic: @Garyfallidis on Horizon API
February 20th, 0800 PT, 1100 ET, 1600 UTC Topic: Workshop prep
March 5th, 0800 PT, 1100 ET, 1600 UTC Topic: @RafaelNH b-tensor encoding

All meetings will be held through Zoom at the following link: https://washington.zoom.us/j/264574770

Meetings will be 60 minutes at the longest.

**Comment by arokem** [comment]
The first meeting will be devoted to a discussion of the preparations for the March DIPY workshop at IU. 
We can also use that meeting to discuss schedule for the rest of the quarter.

**Comment by Garyfallidis** [comment]
Hi @arokem. The first date I think is incorrect. Jan 11th is a Saturday right? You mean Jan 9th?

**Comment by arokem** [comment]
I do in fact! Thanks for pointing that out. Now edited at the top as well. That's _this Thursday_ 1600 UTC.

**Comment by Garyfallidis** [comment]
Okay great, thanks for correcting this. Can you please send emails to the people as the first meeting is very soon.

**Comment by arokem** [comment]
Reminder: tomorrow's DIPY online open lab meeting will take place at 0800 PT, 1100 ET, 1600 UTC. 

I will give an overview of [pyAFQ](https://yeatmanlab.github.io/pyAFQ/)

The link for the call is: https://washington.zoom.us/j/264574770

**Comment by RafaelNH** [comment]
Hello. Was today's meeting cancelled?

**Comment by arokem** [comment]
Yes. Apparently @skoudoro and @Garyfallidis had some conflict, so they couldn't make it. Sorry about that!

**Comment by skoudoro** [comment]
Really sorry about that!

**Comment by arokem** [comment]
Reminder: the meeting about workshop prep will take place in about 10 hours (8 AM PT,  11 AM ET, 16:00 UTC). Here's the link:  https://washington.zoom.us/j/264574770

**Comment by skoudoro** [comment]
Hi @arokem,

I think it is time to restart a new series! I start to miss our DIPY meeting 😄 !  Let's try to plan from April until May.

What do you think?

**Comment by arokem** [comment]
Let's do it! Any objections to keeping this day/time? It seems to work OK for the time-zones we tend to be spread across.

**Comment by skoudoro** [comment]
+1 for keeping the same

**Comment by arokem** [comment]
Superseded by #2109
